website/versioned_docs/version-0.4.x/set-up-your-environment.mdx

```diff
@@ -484,14 +501,142 @@ IMPORTANT: Ceramic API running on 0.0.0.0:7007
+      section at the top.
+    </div>
+  </div>
+</details>
+
+<details>
+  <summary>Which node version is preferred?</summary>
+  <div>
+    <div>
+      We have seen the best results using Node.js v16. Earlier or later versions can cause issues
+      for some users. While we're working on eliminating the issues, it's best to use Node v16 for
+      now.
+    </div>
+  </div>
+</details>
+<details>
+  <summary>How long does it take to install the packages?</summary>
+  <div>
+    <div>
+      Installing everything (either with wheel or JavaScript packages) takes usually between 2 and 5
+      minutes. Throughout the guide above you can find what kind of output you should be looking for
+      to know that everything was installed correctly.
+    </div>
+  </div>
+</details>
+<details>
+  <summary>Where in the system do I run all of the commands?</summary>
+  <div>
+    <div>
+      Sometimes, especially when using JavaScript package managers to install Ceramic and ComposeDB,
+      it's easy to forget that you need to run all of the commands in the app's directory. With
+      JavaScript package managers you need to open 2-3 terminal windows and run different commands,
+      so it's easy to miss that you can be in a wrong directory. Please make sure you run all the
+      commands where they're supposed to run.
+    </div>
+  </div>
+</details>
+<details>
+  <summary>Where can I find a Ceramic node configuration file, daemon.config.json?</summary>
+  <div>
+    <div>
+      When installing ComposeDB with JavaScript package managers, at some point you need to edit
+      your Ceramic node config file. By default, it's in your home directory, in .ceramic folder
+      (*not* in the app directory). It's easy to miss this details so please check the path. This
+      command should take you to the right directory: cd ~/.ceramic
+    </div>
+  </div>
+</details>
+<details>
+  <summary>How to restart a node after stopping it?</summary>
+  <div>
+    When you use wheel to install Ceramic and ComposeDB, it takes care of the whole installation
+    process. But please note that wheel is just an installer, not a node launcher. If you want to
+    launch Ceramic and ComposeDB again, after you have stopped it, you need to launch Ceramic daemon
+    again and then launch ComposeDB.
+  </div>
+  <div>
+    You can launch Ceramic daemon by running the following command: ceramic daeomn
+    --network=InMemory
+  </div>
+  <div>You can launch ComposeDB by running the command: composedb</div>
+  <div>
+    More on all of the composedb command options can be found in "2. Create your composite" section
+    of this Getting Started guide.
+  </div>
+</details>
+<details>
+  <summary>How do I interact with the data once the node is running?</summary>
+  <div>
+    <div>
+      You interact with data through a GraphQL Server. You can find all the details on how to set it
```

Comment:
Technically it depends. GraphQL Server is one of the options to interact with the data. Developers can also use ComposeDB client if they prefer, especially when building their apps: https://composedb.js.org/docs/0.4.x/guides/composedb-client/javascript-client 

Maybe let's frame this paragraph as "the easiest way to interact with data" and "the easiest way to get started" is using GraphQL server.

Reply:
Updated.